core/vm/contracts.go

```diff
@@ -166,6 +190,7 @@ func init() {
 	}
 	for k := range PrecompiledContractsCancun {
 		PrecompiledAddressesCancun = append(PrecompiledAddressesCancun, k)
+		log.Info("PrecompiledAddressesCancun", "k", k)
```

Comment:
@kingster-will Do we need this log?

Reply:
accidentally checked-in. removed the test logs.